index.html

```diff
@@ -3484,6 +3484,29 @@ <h3>Media Types</h3>
 those other syntaxes to be transformed to the `application/vc+ld+json`
 media type.
         </p>
+
+        <p>
+Additional `application/vc+` media type transformation specifications can be
```

Comment:
```suggestion
Additional media type transformation specifications can be
```

I'm wondering if transform specifications need to have a `application/vc+` prefix? It would probably help identify that the serialization is "in the class of VCs"... but, wonder if that's too limiting? For example, what if you could just transform an `application/foo` into an `application/vc+ld+json` because the former spec has a section on how to do the transform. Do the media types really have to start with `application/vc+`

Reply:
If we provide this guidance, don't we have to reserve `application/vc` by itself?